node-mysql-wrapper/node-mysql-wrapper-tests.ts

```diff
@@ -1,18 +1,24 @@
 ///<reference path='./node-mysql-wrapper.d.ts' />
 
-import wrapper = require("node-mysql-wrapper");
-var db = wrapper("mysql://kataras:pass@127.0.0.1/taglub?debug=false&charset=utf8");
+var wrapper = require("node-mysql-wrapper");
```

Comment:
Change it to import if you want, both ways are working I don't know which is the best for simple test file